docs/introduction/architecture.md

```diff
@@ -3,51 +3,51 @@ title: Shard Architecture
 
 ## Shard Token
 
-The private shard token is only used for consensus, to ensure that validators with a certain stake get the ability to verify and propose blocks. The token can be customized to be a CBDC, but by default it will be just used for consensus, to create accounts and to be attached for gas fees.
-  
+The private shard token serves one main purpose: achieving consensus within the shard. It allows validators with a stake in the network to verify and propose blocks. The token is primarily used for consensus, creating accounts, and attaching gas fees. Although it can be customized to act as a Central Bank Digital Currency (CBDC), its default usage is limited to these specific functions
+
 ## Gas Fees
 
-Due to the fact that by default the network’s token doesn’t have any monetary value, no gas fees will be paid on the network. The current design still has limitations on execution per contract call of 300 TGas, but all gas attached will be refunded. 
+Since the network's token does not possess any monetary value by default, no gas fees are paid within the network. However, there are still limitations on execution per contract call, currently set at 300 TGas. Any gas attached to transactions will be refunded.
 
 ## Validators
 
-Private shard validators will own some percentage of the stake (tokens) in the shard defined in the genesis of the network. The tokens are used for consensus only, and do not have any real monetary value. This is the default behavior and it is possible to create networks with actual monetary value if needed in cases of CBDC and similar. To run a minimal private shard you could run only one node controlling the network, but we recommend to run a minimum of four validators, as the network will halt if ⅓ of the nodes go down. This gives minimal redundancy and we recommend running as many validators as possible.
-For example, if a private shard is run by 100 validators owned by 100 different companies, it can be configured so that every single one of them has 1 percent of the total stake distributed. In this case every company will produce 1% of the blocks and more than 66% of the companies will need to behave maliciously and collude to corrupt the shard and 33% + 1 to stall the network.
-In the future we plan to implement different validator staking mechanisms. Being an owner of an NFT could be an entry ticket to become an validator in the shard. The NFT can represent anything from ownership of a game to having gone through KYC.
+Private shard validators hold a portion of the tokens allocated at the beginning of the shard. These tokens are solely used for achieving consensus and do not possess any real monetary value. By default, they are designed for networks without actual monetary transactions. However, it is possible to create networks with monetary value, such as in the case of a Central Bank Digital Currency (CBDC).
```

Comment:
allocated at the shard creation.

Reply:
At least one validator is needed for the private shard to work properly, but it is recommended to have at least four validators for redundancy and decentralisation